internal/core/input.rs

```diff
@@ -685,6 +683,18 @@ pub fn process_mouse_input(
     }
     send_exit_events(&mouse_input_state, &mut result, mouse_event.position(), window_adapter);
 
+    if let MouseEvent::Wheel { position, .. } = mouse_event {
+        if r.has_aborted() {
```

Comment:
`has_aborted()` is a funny way of reading "the event was accepted". I understand that it means (visitor aborted), but I'm also not sure how to make it more readable. The comment you added is good enough I guess :)